src/editor/EditorWindow.cpp

```diff
@@ -229,6 +230,17 @@ EditorWindow::EditorWindow(QWidget *parent)
   connect(ui->actionCopyOptions, &QAction::triggered, [this]() {
     m_copy_options_dialog->setVisible(!m_copy_options_dialog->isVisible());
   });
+
+  if(Settings::ShowLandingPage::get())
+    {
+      m_welcome_dialog = new WelcomeDialog;
+      m_welcome_dialog->setWindowTitle(this->windowTitle());
+
+      connect(m_welcome_dialog, &WelcomeDialog::newSelected, [this]() { Host(HostSetting::NewFile); });
+      connect(m_welcome_dialog, &WelcomeDialog::openSelected, [this]() { Host(HostSetting::LoadFile); });
+      connect(m_welcome_dialog, SIGNAL(connectSelected()), this, SLOT(connectToHost()));
```

Comment:
nit, could you use the function pointer syntax that the other conections use? E.g., &WelcomeDialog::conectSelected and &EditorWindow::connectToHost instead of SIGNAL(...)?

Reply:
done